anker/src/metrics.rs

```diff
@@ -0,0 +1,39 @@
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use lido::token::StLamports;
+use serde::Serialize;
+
+use crate::token::{self, MicroUst};
+
+#[repr(C)]
+#[derive(
+    Clone, Debug, Default, BorshDeserialize, BorshSerialize, BorshSchema, Eq, PartialEq, Serialize,
+)]
+pub struct Metrics {
+    /// Total swapped amount of StSOL to UST.
+    #[serde(rename = "swaped_rewards_total_st_lamports")]
+    pub swapped_rewards_st_sol_total: StLamports,
+
+    /// Total swapped amount of UST to StSOL.
```

Comment:
Nit: not really UST swapped for stSOL, right? Maybe write “Total amount of UST received through swaps”?